pkg/loader/compose/v3.go

```diff
@@ -294,6 +294,20 @@ func dockerComposeToKomposeMapping(composeObject *types.Config) (kobject.Kompose
 			serviceConfig.Replicas = int(*composeServiceConfig.Deploy.Replicas)
 		}
 
+		// placement:
+		placement := make(map[string]string)
+		for _, j := range composeServiceConfig.Deploy.Placement.Constraints {
+			p := strings.Split(j, " == ")
+			if p[0] == "node.hostname" {
+				placement["kubernetes.io/hostname"] = p[1]
+			} else if p[0] == "engine.labels.operatingsystem" {
+				placement["beta.kubernetes.io/os"] = p[1]
+			} else {
+				log.Warn(p[0], " constraints in placement is not supported")
```

Comment:
So just a second review, but maybe we should add that only `node.hostname` and `engine.labels.operatingsystem` is only supported as a constraint in the warning.